src/main/java/com/uditagarwal/algoritms/DoublyLinkedList.java

```diff
@@ -82,4 +70,24 @@ public void addNodeAtLast(DoublyLinkedListNode<E> node) {
         addNodeAtLast(newNode);
         return newNode;
     }
+
+    public boolean isItemPresent() {
+        return dummyHead.next != dummyTail;
+    }
+
+    public DoublyLinkedListNode getFirstElement() throws NoSuchElementException {
```

Comment:
Actually, we are referring element as path content of a node, so lets rename these methods to `getFirstNode` and `getLastNode`